packages/py3_ply.rb

```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Py3_ply < Package
+  description 'Python Lex/Yacc is an implementation of lex and yacc parsing tools for Python.'
+  homepage 'https://www.dabeaz.com/ply/index.html'
+  version '3.11'
+  license 'BSD'
+  compatibility 'all'
+  source_url 'https://files.pythonhosted.org/packages/e5/69/882ee5c9d017149285cab114ebeab373308ef0f874fcdac9beb90e0ac4da/ply-3.11.tar.gz'
```

Comment:
Can we use `https://www.dabeaz.com/ply/ply-3.11.tar.gz` instead?

Reply:
No, as tarballs from pypi.org have special metadata not found in normal tarballs, hence why I try to use git repositories or pypi tarballs wherever possible.

Reply:
also that tarball seems to have a broken setup.py file